From d71205d14c74516925c0e837ba02135f36f084c6 Mon Sep 17 00:00:00 2001
From: Natanael Copa <ncopa@alpinelinux.org>
Date: Mon, 6 Jun 2011 14:55:19 +0000
Subject: [PATCH 2/2] setup-gparted-desktop: new setup script

The idea is that you quick and dirty can get gparted up and running
---
 Makefile                 |    1 +
 setup-gparted-desktop.in |   42 ++++++++++++++++++++++++++++++++++++++++++
 2 files changed, 43 insertions(+), 0 deletions(-)
 create mode 100644 setup-gparted-desktop.in

diff --git a/Makefile b/Makefile
index 91f0835..0427b1b 100644
--- a/Makefile
+++ b/Makefile
@@ -25,6 +25,7 @@ SBIN_FILES	:= lbu\
 		setup-bootable\
 		setup-timezone\
 		setup-xorg-base\
+		setup-gparted-desktop\
 		update-conf
 
 BIN_FILES	:= uniso
diff --git a/setup-gparted-desktop.in b/setup-gparted-desktop.in
new file mode 100644
index 0000000..8bd198e
--- /dev/null
+++ b/setup-gparted-desktop.in
@@ -0,0 +1,42 @@
+#!/bin/sh
+
+setup-xorg-base gparted xinit aterm openbox ttf-freefont $@
+
+# create openbox menu
+mkdir -p ~/.config/openbox
+cat >~/.config/openbox/menu.xml<<EOF
+<?xml version="1.0" encoding="UTF-8"?>
+
+<openbox_menu xmlns="http://openbox.org/3.4/menu">
+<menu id="root-menu" label="Openbox 3">
+  <separator label="Applications" />
+  <item label="GParted">
+    <action name="Execute">
+      <command>gparted</command>
+    </action>
+  </item>
+  <item label="aterm">
+    <action name="Execute">
+      <command>aterm</command>
+    </action>
+  </item>
+  <separator />
+  <item label="Log Out">
+    <action name="Exit">
+      <prompt>yes</prompt>
+    </action>
+  </item>
+</menu>
+</openbox_menu>
+EOF
+
+# create xinitrc
+cat >~/.xinitrc <<EOF
+gparted &
+exec openbox-session
+EOF
+
+echo ""
+echo " To start GParted run: startx"
+echo ""
+
-- 
1.7.5.4

